cornflow/schemas/schema_manager.py

```diff
@@ -0,0 +1,368 @@
+"""
+Class to help create and manage data schema and to validate json files.
+"""
+
+
+import json
+import re
+from jsonschema import validate
+from copy import deepcopy
+from genson import SchemaBuilder
+from ..schemas.schema_dict_functions import gen_schema, ParameterSchema, sort_dict
+from .constants import JSON_TYPES
+
+
+class SchemaManager:
+    
+    def __init__(self, schema_path=None):
+        """
+        Class to help create and manage data schema.
+        Once a schema is loaded, allow the validation of data.
+        """
+    
+        self.jsonschema = {}
+        self.types = JSON_TYPES
+        self.schema_dict = {"DataSchema": {}}
+        self.flask_schemas = {}
+        self.draft_schema = {}
+        if schema_path is not None:
+            self.load_schema(schema_path)
+    
+    def get_jsonschema(self):
+        """
+        Return a copy of the stored jsonschema.
+        """
+        return deepcopy(self.jsonschema)
+    
+    def is_schema_loaded(self):
+        """
+        Check if a schema is loaded.
+        """
+        loaded = len(self.jsonschema) > 0
+        if not loaded:
+            raise Warning("No jsonschema has been loaded in the SchemaManager class")
+        return loaded
+    
+    def validate_data(self, data_path):
+        """
+        Validate a json file according to the loaded jsonschema.
+
```

Comment:
same as the other comment. Better to have two functions (one with a dictionary, one with a path)

Reply:
done